Library/Homebrew/os/mac/version.rb

```diff
@@ -7,13 +7,15 @@ module OS
   module Mac
     class Version < ::Version
       SYMBOLS = {
-        big_sur:     "11.0",
-        catalina:    "10.15",
-        mojave:      "10.14",
-        high_sierra: "10.13",
-        sierra:      "10.12",
-        el_capitan:  "10.11",
-        yosemite:    "10.10",
+        big_sur:       "11.0",
+        catalina:      "10.15",
+        mojave:        "10.14",
+        high_sierra:   "10.13",
+        sierra:        "10.12",
+        el_capitan:    "10.11",
+        yosemite:      "10.10",
+        mavericks:     "10.9",
+        mountain_lion: "10.8",
```

Comment:
if we decide to go this way, I will add a comment to keep old versions (`mavericks` was removed from the list in https://github.com/Homebrew/homebrew-core/commit/b58fa4ebb1e6b918d8556cba629cb55b79d423f5#diff-c6bee041a512d5f6f87141846ac1e88b, for example).